### PR TITLE
Add catchError to stages of node image building pipeline

### PR DESCRIPTION
**Pull request by namnx228**
Currently, whenever a stage in node image and openstack image jobs is failed, the whole build stops. This PR helps to continue the build even when a stage is failed, and mark the build as `FAILURE` eventually.

### CODE REVIEW

File: ci/jobs/openstack_image_building.pipeline [hunk 1]
```diff
@@ -51,58 +51,65 @@ pipeline {
...
+                registry.nordix.org/airship/image-builder \
+                /data/ci/images/gen_base_ubuntu_image.sh \
+                /data/id_rsa_airshipci 1"
 
+            }
           }
-        }
-      }
+        } 
+       }
```
Comment (smoshiur1237) [inline ci/jobs/openstack_image_building.pipeline:92]: There are some indentations in the file. can you fix please?

Comment (namnx228) [reply ci/jobs/openstack_image_building.pipeline]: Fixed! Thanks

**Pull request merged**